plugins/callback/opentelemetry.py

```diff
@@ -131,7 +139,11 @@ def __init__(self, uuid, name, path, play, action, args):
         if sys.version_info >= (3, 7):
             self.start = time.time_ns()
         else:
-            self.start = _time_ns()
+            # Support versions older than 3.7 with opentelemetry-api > 1.12
+            if OTEL_LIBRARY_TIME_NS_ERROR:
+                self.start = int(time.time() * 1e9)
+            else:
+                self.start = _time_ns()
```

Comment:
How about moving this whole double-`if` into a new function? Especially since it needs to be called twice (here and below)?